src/antq/core.clj

```diff
@@ -151,13 +181,16 @@
         (log/info))))
 
 (defn- assoc-latest-version
-  [dep options]
+  [dep options forced-artifacts]
   (let [vers (cond->> (:_versions dep)
                (not (ver/under-development? (:version dep)))
                (drop-while ver/under-development?))
         vers (remove-skipping-versions vers dep options)
         latest-version (first vers)]
-    (assoc dep :latest-version latest-version)))
+    (if-let [forced-version (and (seq forced-artifacts)
+                                 (forced-version dep forced-artifacts))]
+      (assoc dep :latest-version forced-version :forced? true)
+      (assoc dep :latest-version latest-version))))
```

Comment:
We are replacing the version number here, but by this point, the retrieval of the actual version number information has already been fetched from maven, github, etc.

If a version number is specified with focus, wouldn't we want to skip retrieving the version number information? (It takes time, and finally, it's not used anyway.)